### PR TITLE
added MITRE ATT&CK tags to all Slack rules

### PR DESCRIPTION
**Pull request by arielkr256**
### Background

Slack rules have the highest volume of critical and high alerts among rules without MITRE ATT&CK tags.

### Changes

* Added MITRE tactic and technique friendly names to Tags and MITRE tactic:technique codes to Reports: MITRE ATT&CK

### Testing

* Metadata changes only

### CODE REVIEW

File: rules/slack_rules/slack_app_added.yml [hunk 1]
```diff
@@ -7,6 +7,11 @@ LogTypes:
   - Slack.AuditLogs
 Tags:
   - Slack
+  - Persistence
+  - Create Account
+Reports:
+  MITRE ATT&CK:
+    - TA0003:T1136
```
Comment (egibs) [inline rules/slack_rules/slack_app_added.yml:14]: Would `T1505` be more applicable here? `1136` also works, but:
```
Adversaries may abuse legitimate extensible development features of servers to establish persistent access to systems. Enterprise server applications may include features that allow developers to write and install software or scripts to extend the functionality of the main application. Adversaries may install malicious components to extend and abuse server applications. 
```

File: rules/slack_rules/slack_private_channel_made_public.yml [hunk 2]
```diff
@@ -7,6 +7,11 @@ LogTypes:
   - Slack.AuditLogs
 Tags:
   - Slack
+  - Defense Evasion
+  - File and Directory Permissions Modification
+Reports:
+  MITRE ATT&CK:
+    - TA0005:T1222
```
Comment (egibs) [inline rules/slack_rules/slack_private_channel_made_public.yml:14]: Would `TA0003:T1098` be more applicable here?
```
... Account manipulation may consist of any action that preserves or modifies adversary access to a compromised account, such as modifying credentials or permission groups. These actions could also include account activity designed to subvert security policies, such as performing iterative password updates to bypass password duration policies and preserve the life of compromised credentials.
```

File: rules/slack_rules/slack_service_owner_transferred.yml [hunk 3]
```diff
@@ -7,6 +7,11 @@ LogTypes:
   - Slack.AuditLogs
 Tags:
   - Slack
+  - Defense Evasion
+  - File and Directory Permissions Modification
+Reports:
+  MITRE ATT&CK:
+    - TA0005:T1222
```
Comment (egibs) [inline rules/slack_rules/slack_service_owner_transferred.yml:14]: Similar comment about using `TA0003` here. I can see `TA0005` being valid as well.

**Review by egibs — COMMENTED** [review]
Overall LGTM; I just had a few clarifying questions.

File: rules/slack_rules/slack_app_added.yml (hunk 1 above)
Comment (arielkr256) [reply rules/slack_rules/slack_app_added.yml]: I like this, updated

File: rules/slack_rules/slack_private_channel_made_public.yml (hunk 2 above)
Comment (arielkr256) [reply rules/slack_rules/slack_private_channel_made_public.yml:14]: This one could fall into several categories, `T1098` makes sense, `T1562` or even `T1567` could also be applicable.

File: rules/slack_rules/slack_service_owner_transferred.yml (hunk 3 above)
Comment (arielkr256) [reply rules/slack_rules/slack_service_owner_transferred.yml:14]: Again several applicable techniques, maybe even `T1531`.

**Review by egibs — APPROVED** [review]

**Pull request merged**